### PR TITLE
Test get latest output artifact

### PR DESCRIPTION
**Pull request by mayabrandi**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -13,23 +13,34 @@
 
 
 class MockProcess():
-    def __init__(self, date_str = '2014-01-28'):
+    def __init__(self, date_str = None, process_type = None):
```
Comment (moonso) [inline tests/conftest.py]: Wouldn't it be good to let the MockProcess have a default date? There are several functions that assume that `artifact.parent_process.date_run` exists and is a string...

File: tests/build/test_lims_utils.py [hunk 2]
```diff
@@ -75,5 +76,38 @@ def test_get_number_of_days():
...
+    process3 = lims._add_process(date3, process_type)
+    artifact1 = lims._add_artifact(process1)
+    artifact2 = lims._add_artifact(process2)
+    artifact3 = lims._add_artifact(process3)
+
+    ##WHEN running _get_latest_output_artifact
+    latest_output_artifact = _get_latest_output_artifact(process_type, lims_id, lims)
+
+    ##THEN latest_output_artifact should be run on 2018-03-01
+    assert latest_output_artifact.parent_process.date_run == date3
```
Comment (moonso) [inline tests/build/test_lims_utils.py]: 👍

File: tests/conftest.py (hunk 1 above)
Comment (mayabrandi) [reply tests/conftest.py]: True

File: tests/build/test_lims_utils.py [hunk 3]
```diff
@@ -75,5 +76,76 @@ def test_get_number_of_days():
...
+    out_art3 = lims._add_artifact(process3)
+
+    ##WHEN running _get_latest_output_artifact
+    latest_output_artifact = _get_latest_output_artifact(process_type, lims_id, lims)
+
+    ##THEN latest_output_artifact should be run on 2018-03-01
+    assert latest_output_artifact.parent_process.date_run == date3
+
+
+def test_get_latest_input_artifact(lims):
```
Comment (mayabrandi) [inline tests/build/test_lims_utils.py]: @moonso I put one more test here. Don't know if you noticed before you approved the PR?

Comment (moonso) [reply tests/build/test_lims_utils.py]: I saw it 👍

**Pull request merged**